src/swap/types.ts

```diff
@@ -219,6 +232,7 @@ export type SwapUnit = {
   balance?: string;
   error?: SwapError;
   loading: boolean;
+  refetch: () => Promise<void>;
```

Comment:
Let's add a line of comment to explain what this does.

The word `refetch` feels very vague and hard to understand in a few weeks from now.

Reply:
using `tokenResponse` instead